Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:lts-alpine as builder
+
+# Install dependencies
+RUN apk add --no-cache git
```

Comment:
Why do you need Git?

Reply:
Needed by since the source image is alpine based image (to reduce image size) which doesn't have `git` installed by default.  This is the error when git is not found.


```51.44 npm notice
51.44 npm notice New patch version of npm available! 10.5.0 -> 10.5.2
51.44 npm notice Changelog: <https://github.com/npm/cli/releases/tag/v10.5.2>
51.44 npm notice Run `npm install -g npm@10.5.2` to update!
51.44 npm notice
51.44 npm ERR! code ENOENT
51.44 npm ERR! syscall spawn git
51.44 npm ERR! path git
51.44 npm ERR! errno -2
51.44 npm ERR! enoent An unknown git error occurred
51.44 npm ERR! enoent This is related to npm not being able to find a file.
51.45 npm ERR! enoent
```